### PR TITLE
security email: discourage use #760

### PR DESCRIPTION
**Pull request by RandyMcMillan**

### CODE REVIEW

**Comment by RandyMcMillan** [comment]
Screen Shot 2021-08-04 at 10 21 54 PM

File: _posts/en/pages/2016-03-12-contact.md [hunk 1]
```diff
@@ -7,17 +7,30 @@ layout: page
...
-To report security issues: <i class="fa fa-fw fa-envelope"></i> security<span style="display:none"></span>@bitcoincore.org (<i style="color:red">not for support</i>).
+<center>
+<hr>
+<i style="font-weight: bold;font-size: 184%;">Bitcoin Core does <span style="color:red">not</span> provide <wbr>technical support.
+</i>
+<i style="font-weight: bold;font-size: medium;"><br>Community support can be found at <a style="color:blue" href="https://bitcoin.stackexchange.com/">bitcoin.stackexchange.com</a></i>
+<hr>
+</center>
+
+<i class="fa fa-fw fa-twitter"></i> <span style="color:blue">Twitter: <a href="https://twitter.com/bitcoincoreorg/">@bitcoincoreorg</a></span>
```
Comment (harding) [inline _posts/en/pages/2016-03-12-contact.md:19]: What's the motivation for this change to the the Twitter line?

**Review by harding — COMMENTED** [review]
Concept ACK.  One question:

**Comment by RandyMcMillan** [comment]
open to suggestions - i just wanted to post something to prompt discussion.

**Comment by TheBlueMatt** [comment]
Honestly we could just drop the email alias on the website and put it on Github only, then link more strongly to the github on the contact page?

**Comment by Zero-1729** [comment]
Agree, hopefully that would solve the spamming issue, assuming those who would see it on Github won't still email it for support.

Alternatively, if dropping the email alias from the site is too destructive a change, the security disclosure bit could be left the way it currently is on the site (small and easy to miss). The stack exchange can be left in bold, as someone desperately seeking help would likely be in a rush for answers and the large community support message in this PR would catch their attention first (which is good). However, seeing another large message for security disclosure might cause them to consider that as an option for getting support as well (even despite the "*not for support*" message, people are just funny like that).

Regardless, I'd prefer just dropping it from the site and moving it to Github only as suggested above.

**Comment by harding** [comment]
@TheBlueMatt 

> Honestly we could just drop the email alias on the website and put it on Github only, then link more strongly to the github on the contact page?

I'm happy to merge any changes wanted by the people on the list, but if moving the disclosure information to a different URL is something you think would work, we could just move it to a different page on the site.

**Comment by TheBlueMatt** [comment]
Oops, to clarify, I’m not advocating moving security disclosure to GitHub issues, I’m suggesting that the only mention of the security disclosure alias be on GitHub issue default text or other similar. It’s just an idea and I’m not on that alias anymore so I can’t speak for anyone :).

**Comment by laanwj** [comment]
Concept ACK. Thank you for working on this. I think this is a good solution to further discourage use of the alias for technical support, that it's only for reporting security issues.

> Alternatively, if dropping the email alias from the site is too destructive a change, the security disclosure bit could be left the way it currently is on the site (small and easy to miss). 

Agree, no need to make the "responsible disclosure" part big. People who find security issues tend to be good at finding semi-hidden links too :smile:

**Comment by RandyMcMillan** [comment]
0aeee59

Screen Shot 2021-09-17 at 1 33 02 AM

**Comment by RandyMcMillan** [comment]
ee2884d

Screen Shot 2021-09-17 at 1 40 37 AM

**Comment by RandyMcMillan** [comment]
1a12a84

Screen Shot 2021-09-17 at 1 47 10 AM

**Comment by RandyMcMillan** [comment]
197d098

Screen Shot 2021-09-17 at 1 48 50 AM

**Comment by RandyMcMillan** [comment]
dea9847

Screen Shot 2021-09-17 at 1 50 18 AM

**Comment by RandyMcMillan** [comment]
Here are a few variations to cherry-pick from...

**Comment by harding** [comment]
Tested ACK dea98475b78ea9fa115bc81634218e9876115adf .  I think the current commit tip is one of the better options, and there's no need to delay this further---we can always tweak things again later.

Thanks @RandyMcMillan for working on this and for providing multiple options!
